README.md

```diff
@@ -72,12 +52,22 @@ C# · Go · Java · JavaScript · JSX · JSON · PHP · Python · Ruby · Scala
 <p align="center">
 See <a href="https://semgrep.dev/docs/supported-languages/">supported languages</a> for the complete list.
 </p>
+<br/>
 
-### Getting started
+### Getting started 🚀
 
-To install Semgrep use Homebrew or pip, or run without installation via Docker:
+1. [Getting started from the CLI](#option-1-getting-started-from-the-cli)
+2. [Getting started from the web UI](#option-2-getting-started-via-the-web-ui-recommended)
 
-```sh
+We recommend starting from the [web UI](#option-2-getting-started-via-the-web-ui-recommended) and then install the CLI because it is simple and allows you to quickly see and manage results, as well as set up CI/CD pipelines with minimal effort. You can also use additional proprietary Semgrep Registry rules and run custom policies from the Semgrep App.
+
+However, you can also start with the CLI directly, even without logging in, but you will lose the benefits mentioned earlier.
+
+### Option 1: Getting started from the CLI
```

Comment:
alt:

### Option 1: Getting started (no login)

1.  Install Semgrep CLI

```
# For macOS
$ brew install semgrep

# For Ubuntu/WSL/Linux/macOS
$ python3 -m pip install semgrep

# To try Semgrep without installation run via Docker
$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
```

2. Go to a repository's root directory and run `semgrep --config auto`. This will scan your project with default settings.